help/view-and-edit-your-message-drafts.md

```diff
@@ -14,24 +15,76 @@ so that you never lose your work. Drafts are saved for 30 days.
 
 {tab|desktop-web}
 
-1. Simply close the compose box. You can hit <kbd>Esc</kbd>, click
-the <i class="fa fa-remove"></i> in the top right corner of the
-compose box, or click on an empty part of the app.
+{!start-composing.md!}
+
+1. Write a message.
+
+1. Click the <i class="fa fa-remove"></i> in the top right corner of the compose
+   box, or click on an empty area of the app to close the compose box and save
+   your message as a draft.
```

Comment:
I prefer phrasing like "Close the compose box. For example, you can click ..." -- because it's important for feeling safe about your content being lost when the compose box is closed due to some more complex interaction, we want to imply really clearly that no matter how the compose box gets closed, as long as the message wasn't sent, your content gets saved as a draft.